src/main/webapp/app/course/learning-paths/participate/lecture-unit/learning-path-lecture-unit-view.module.ts

```diff
@@ -0,0 +1,33 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { LearningPathLectureUnitViewComponent } from 'app/course/learning-paths/participate/lecture-unit/learning-path-lecture-unit-view.component';
+import { ArtemisLectureUnitsModule } from 'app/overview/course-lectures/lecture-units.module';
+import { ArtemisSharedModule } from 'app/shared/shared.module';
+import { Authority } from 'app/shared/constants/authority.constants';
+import { UserRouteAccessService } from 'app/core/auth/user-route-access-service';
+
+const routes: Routes = [
+    {
+        path: '',
+        component: LearningPathLectureUnitViewComponent,
+        data: {
+            authorities: [Authority.USER],
+            pageTitle: 'overview.learningPath',
+        },
+        canActivate: [UserRouteAccessService],
```

Comment:
Be aware that you do not always use this service in your newly introduced routes (in another file).